build-support/bin/shellcheck.py

```diff
@@ -22,11 +22,11 @@ def ensure_shellcheck_installed() -> None:
 
 
 def run_shellcheck() -> None:
-  targets = glob("./build-support/bin/**/*.sh", recursive=True) + [
+  targets = glob("./build-support/bin/*.sh") + glob("./build-support/bin/native/*.sh") + [
```

Comment:
Why not recursive?

Reply:
Because `build-support` includes downloaded and generated files, e.g.

```
build-support/virtualenv.dist/tests/test_activate.sh
build-support/virtualenv.dist/virtualenv_embedded/activate.sh
```

Reply:
Ugh, we should really move the virtualenvs outside of build-support... But in the mean time, I'd be tempted to keep it as a recursive glob, and add explicit excludes, to future-proof for when we add more directories in the future :)